src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/ProtocolGasDataRows.js

```diff
@@ -0,0 +1,94 @@
+
+import React, { useState, useMemo, useEffect } from "react";
+import { Button } from "@trussworks/react-uswds";
+
+import { getQAProtocolGas } from "../../../utils/api/qaCertificationsAPI";
+import QADataTableRender from "../../QADataTableRender/QADataTableRender";
+import { Preloader } from "@us-epa-camd/easey-design-system";
+
+const tableLabel = 'Protocol Gas'
+
+const ProtocolGasDataRows = ({
+  user,
+  data
+}) => {
+  const { locationId, id } = data;
+  const [protocolGasData, setProtocolGasData] = useState()
+  const [loading, setLoading] = useState(false)
+  const columns = [
+    'Gas Level Code',
+    'Gas Type Code',
+    'Cylinder ID',
+    'Vendor ID',
+    'Expiration Date',
+  ];
+
+  useEffect(() => {
+    const fetchProtocolGas = async () => {
+      setLoading(true)
+      const resp = await getQAProtocolGas(locationId, id)
```

Comment:
can you add a validation to the API response to check if the response is a success, and add a handler if not.  Also this way when you run the associated unit test, it won't complain.